protocol/consistencychecks_test.go

```diff
@@ -3,10 +3,34 @@ package protocol
 import "testing"
 
 var (
-	uname = "alice"
-	key   = []byte("key")
+	key = []byte("key")
 )
 
+func doRequestAndVerify(d *ConiksDirectory,
```

Comment:
I'm not sure how I feel about refactoring all the functionality into a single function. Now it's more difficult to follow what each test is really doing.